crates/hc_sandbox/Cargo.toml

```diff
@@ -34,6 +34,7 @@ holo_hash = { version = "^0.6.0-dev.0", path = "../holo_hash", features = [
   "kitsune2",
 ] }
 holochain_chc = { version = "^0.3.0-dev.0", path = "../holochain_chc", optional = true }
+holochain_client = { path = "../client" }
```

Comment:
Do we need a version number here? I'm not sure how they're getting updated because the rust client has a different minor version than the other holochain crates and I'm assuming there's some automation in place to update these versions numbers?